src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CalibrationTestWindow.cs

```diff
@@ -35,7 +35,8 @@ internal class CalibrationTestWindow : CompositorWindowBase<CalibrationTestWindo
 
         private static readonly string holographicCameraIPAddressKey = $"{nameof(CalibrationTestWindow)}.{nameof(holographicCameraIPAddress)}";
         private static readonly string calibrationPlaybackIndexFilePathPreferenceKey = $"{nameof(CalibrationTestWindow)}.{nameof(indexFilePath)}";
-        private static readonly string calibrationPlaybackCalibrationFilePathPreferenceKey = $"{nameof(CalibrationTestWindow)}.{nameof(calibrationFilePath)}";
+        private static readonly string calibrationPlaybackCalibrationFilePathPreferenceKey = $"{nameof(CalibrationTestWindow)}.{nameof(calibrationFilePath)}";
+        private static readonly string markerSizeForPlaybackPreferenceKey = $"{nameof(CalibrationTestWindow)}.MarkerSizeForPlayback";
```

Comment:
>MarkerSizeForPlayback [](start = 110, length = 21)

Could make this '{nameof(markerSizeForPlayback)}' to match the pattern the other fields use